doc/performance-testing/cloud-run/0001_stress-test-december-2021.md

```diff
@@ -0,0 +1,86 @@
+# Stress Test December 2021
+This is an initial stress test to asses the performance of runner after the switch from kubernetes to cloud run. Below are the initial settings we used:
+
+
+## Runner settings
```

Comment:
Is there any other settings that would help here, image tag, cpu, memory etc

Reply:
Added the additional vars that we had set for runner from our pipeline 👍

Reply:
Can we add the image tag/runner version used.